components/CommentModal/CommentModal.js

```diff
@@ -136,19 +60,9 @@ See ${webSiteBillAddress} for details.`
     setIsPublishing(true)
     await edit.saveDraft.execute(testimony)
     await edit.publishTestimony.execute()
-
-    if (checkedSendToYourLegislators) {
-      window.open(emailCommandToMyLegislators) // allow user to send a formatted email using their email client
-    }
-    if (checkedSendToCommittee && (houseChairEmail || senateChairEmail)) {
-      window.open(emailCommandToCommitteeChairs) // allow user to send a formatted email using their email client
-    }
-    if (checkedTweet) {
-      window.open(tweet)
-    }
-
-    handleCloseTestimony()
+    setShowPostSubmitModal(true)
     setIsPublishing(false)
+    // handleCloseTestimony() is in postSubmitModal
```

Comment:
I think we should close the testimony modal when we open the post submit modal, since the user is done interacting with it.